plugins/BEdita/API/src/Controller/LoginController.php

```diff
@@ -82,6 +82,11 @@ public function login()
     {
         $this->request->allowMethod('post');
 
+        if (!empty($this->request->data)) {
```

Comment:
I would do
`if (!empty($this->request->data['password']))`
to avoid errors on missing password in POST